modules/customize-certificates-manually-rotate-service-ca.adoc

```diff
@@ -6,12 +6,7 @@
 = Manually rotate the service CA certificate
 
 The service CA is valid for one year after {product-title} is
```

Comment:
This sentence is now wrong on 4.2+.  I would say "The service CA is valid for one year and automatically refreshed.  Follow these steps to manually refresh the service CA."

Reply:
I think it will in the future be wrong on 4.1+ (we're intending to backport automated rotation 
to 4.1 and 4.2 afaik).

Reply:
Yes this is temporarily removing incorrect information about the current behavior. Will be adding in auto rotation docs in a future PR once it is released in the z-stream.